src/main/java/mekanism/common/integration/OreDictManager.java

```diff
@@ -416,20 +418,6 @@ public static void init(RegistryEvent.Register<IRecipe> event) {
                   new ItemStack(MekanismItems.OtherDust, 9, 3), new GasStack(MekanismFluids.Hydrogen, 1000), 0, 900);
         }
 
-        for (ItemStack coal : OreDictionary.getOres("dustCoal", false)) {
-            RecipeHandler.addPRCRecipe(coal, new FluidStack(FluidRegistry.WATER, 100), new GasStack(MekanismFluids.Oxygen, 100),
-                  new ItemStack(MekanismItems.OtherDust, 1, 3), new GasStack(MekanismFluids.Hydrogen, 100), 0, 100);
-        }
-
-        for (ItemStack coal : OreDictionary.getOres("dustCharcoal", false)) {
-            RecipeHandler.addPRCRecipe(coal, new FluidStack(FluidRegistry.WATER, 100), new GasStack(MekanismFluids.Oxygen, 100),
-                  new ItemStack(MekanismItems.OtherDust, 1, 3), new GasStack(MekanismFluids.Hydrogen, 100), 0, 100);
-        }
-
-        for (ItemStack sulfur : OreDictionary.getOres("sulphur", false)) {
-            RecipeHandler.addEnrichmentChamberRecipe(sulfur, new ItemStack(Items.GUNPOWDER));
-        }
-
         for (ItemStack sawdust : OreDictionary.getOres("dustWood", false)) {
             ItemStack dust = StackUtils.size(sawdust, 8);
             //TODO: 1.14 evaluate adding a charcoal dust item to Mekanism, and if so use that instead of charcoal here
```

Comment:
Should i change that Enrichment Chamber Recipe to use dustCharcoal instead of Charcoal if available?

Reply:
that one in the line under this you can't see here because it is unchanged and can't be used for a comment.

Reply:
and if yes should it be like
````java
    if(oreDict.size() > 0) {
        RecipeHandler.addEnrichmentChamberRecipe(dust, oreDict.get(0));
    } else {
        RecipeHandler.addEnrichmentChamberRecipe(dust, new ItemStack(Items.COAL, 1, 1));
    }
````
or like
````java
    ItemStack charcoal = new ItemStack(Items.COAL, 1, 1);
    if(oreDict.size() > 0) {
        charcoal = oreDict.get(0);
    }
    RecipeHandler.addEnrichmentChamberRecipe(dust, charcoal);
````
?

Reply:
No, leave it as coal for now. Recipes shouldn't really change just because another mod adds an item

Reply:
ok ill let it keep using charcoal.